source/service-standards.html.md.erb

```diff
@@ -0,0 +1,99 @@
+---
+title: Service Standards
+last_reviewed_on: 2021-09-06
+review_in: 3 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+This documentation describes the standards we adhere to when writing services.
+
```

Comment:
One change required - needed to remove the section heading because it was generating an invalid link that was being rejected by the link checker.